index.html

```diff
@@ -242,6 +243,7 @@ <h3>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy for Verifiable Credentials</li>
             <li>An API for Verifiable Credential Exchange</li>
+            <li>Multilingual Text Mapping Data Model</li>
```

Comment:
Not quite clear what this means. It was clear when @shigeya described it on the call.

Reply:
Thanks. that's clearer.